lib/rust/ensogl/core/src/display/object.rs

```diff
@@ -23,3 +27,78 @@ pub mod traits {
     pub use super::Object as TRAIT_Object;
     pub use super::ObjectOps as TRAIT_ObjectOps;
 }
+
+
+
+// =========================
+// === InstanceWithLayer ===
+// =========================
+
+/// A display object with a layer. When [`Instance`] is added to a scene layer, the `layer`
+/// is attached to that new layer using [Layer::add_sublayer]. Likewise, when the instance is
```

Comment:
Again, in my previous review, I made a comment about this doc and it is not fixed. `When [`Instance`] is added to a scene layer, the `layer` is attached to that new layer` - why do we assume here that a "new" layer is being created? Which layer is "new" in this sentence? This is another place that makes this PR process extremely long. We've been already discussing that both textually and during the interactive call.

Reply:
I missed the second `new` word in this sentence while editing it 🤷🏻‍♂️